metabase/metabase
# Migrate NativeQueryBuilder and ExpressionEditor to JS colors

Epic https://github.com/metabase/metabase/issues/23954

How to test ExpressionEditor:
- New -> Question
- Add expression
- Make sure the syntax highlighting works

How to test NativeQueryBuilder:
- New -> SQL Query
- Make sure the SQL syntax highlighting works
